evap/evaluation/tests/test_models.py

```diff
@@ -467,32 +467,32 @@ def test_no_login_url_when_delegates_in_cc(self):
         self.user.delegates.add(self.other_user)
         EmailTemplate.send_to_users_in_courses(self.template, [self.course], EmailTemplate.CONTRIBUTORS, use_cc=True, request=None)
         self.assertEqual(len(mail.outbox), 2)
-        self.assertFalse("loginkey" in mail.outbox[0].body)  # message does not contain the login url
-        self.assertTrue("loginkey" in mail.outbox[1].body)  # separate email with login url was sent
+        self.assertFalse("key" in mail.outbox[0].body)  # message does not contain the login url
```

Comment:
i would check for a longer string, this substring could also appear in the default email templates.

Reply:
The template is generated above in the test class
`        cls.template = mommy.make(EmailTemplate, body="{{ login_url }}")`

However, i see your point. Essentially, one body is empty, the other is something like "https://example.com/key/144875930" if overriding the PAGE_URL setting. Would you rather test for emptiness, for a part of the url or for the full url?

Reply:
i think the complete url, but i have no explanation for that other than that everything else feels even more weird to me. we would need to specify the domain setting for that, but i think that's fine.